docs/development/setup-advanced.md

```diff
@@ -373,6 +374,34 @@ proxy in the environment as follows:
  yarn config set https-proxy http://proxy_host:port
  ```
 
+## Installing directly on cloud9
+This section documents how to setup zulip dev environment on cloud9 workspace.
+If you don't have a cloud9 account signup [here](https://aws.amazon.com/cloud9/)
+although new AWS cloud9 is not free. And the zulip dev env may work with zulip-cloud9 package.
+
+* Create a Workspace, and select the blank template.
+* Resize the workspace to be 1GB of memory and 4GB of disk space. (This is under free limit)
```

Comment:
Does the Zulip dev environment actually work in 1GB without OOM kills?  I would have thought we needed like 1.5GB on a Linux VM, but I guess partially that's because the OS has some overhead; it's possible Cloud9's architecture changes these numbers (I don't know enough about it to be sure).

Reply:
Yeah, i never faced any issues with 1GB memory, in one of them i have (768 MB) works just fine. Its just that it takes 1-2 minutes longer to start the dev server compared to droplet but otherwise i think works pretty well.

Reply:
OK, good to know.